app/src/main/java/com/vultisig/wallet/app/activity/MainActivity.kt

```diff
@@ -42,10 +46,17 @@ internal class MainActivity : AppCompatActivity() {
                     }
                 }
 
-                SetupNavGraph(
-                    navController = navController,
-                    startDestination = screen,
-                )
+                Box {
+                    SetupNavGraph(
+                        navController = navController,
+                        startDestination = screen,
+                    )
+
+                    SnackbarHost(
+                        modifier = Modifier.align(Alignment.BottomCenter),
+                        hostState = mainViewModel.snakeBarHostState
+                    )
+                }
```

Comment:
I want to show success message snackbar while navigating from backup screen to vault settings screen
if snackbar is shown in backup screen and navigating from it, snackbar display cancelled.
so i delegate its display to their common parent. 
in other word, we can show snackbar independent from navigating.
also i do not use scaffold because it force me to use its padding. but in some pages scaffold padding is handled. (i can ignore its force by using from ignore annotation)